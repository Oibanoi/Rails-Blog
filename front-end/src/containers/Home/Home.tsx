import { Typography } from 'antd';
import { t } from 'helpers/i18n';

const { Text } = Typography;

const Home: React.FC = () => {
  return <Text>{t('Home')}</Text>;
};

export default Home;
