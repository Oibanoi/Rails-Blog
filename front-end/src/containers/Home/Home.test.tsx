import { render } from '@testing-library/react';
import Home from './Home';

it('Renders correctly', () => {
  const { container } = render(<Home />);

  expect(container).toMatchSnapshot();
});
