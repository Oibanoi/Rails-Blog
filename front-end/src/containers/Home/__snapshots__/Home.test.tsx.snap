// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <span
    class="ant-typography css-dev-only-do-not-override-pr0fja"
  >
    Trang chá»§
  </span>
</div>
`;
